seeders/20180219074913-ActionsSeed.js

```diff
@@ -6,8 +6,58 @@ module.exports = {
     return queryInterface.bulkInsert('Actions', [
 
       { name: "HTML boilerplate", url: "https://www.sitepoint.com/introduction-html5-boilerplate/" , criterionId: 10000, createdAt: new Date(), updatedAt: new Date(), id: 10000 },
-      { name: "CSS", url: "https://github.com/twbs/bootstrap" , criterionId: 10000, createdAt: new Date(), updatedAt: new Date(), id: 10002 },
+      { name: "Writing HTML", url: "https://www.w3schools.com/html/" , criterionId: 10001, createdAt: new Date(), updatedAt: new Date(), id: 10001 },
+      { name: "Writing HTML", url: "https://www.w3schools.com/html/html_basic.asp" , criterionId: 10001, createdAt: new Date(), updatedAt: new Date(), id: 10002 },
+      { name: "Writing HTML", url: "https://www.codecademy.com/courses/web-beginner-en-HZA3b/0/1?curriculum_id=50579fb998b470000202dc8b" , criterionId: 10002, createdAt: new Date(), updatedAt: new Date(), id: 10003 },
+      { name: "Writing HTML", url: "https://www.w3schools.com/html/html_styles.asp" , criterionId: 10002, createdAt: new Date(), updatedAt: new Date(), id: 10004 },
```

Comment:
These are for styles.  This has nothing to do with memorizing tags. 

Here is a good resource example... https://www.youtube.com/watch?v=QZXDAGwD8Ng